File: MEI/Hasil/CODING/ID GAME/ML.py
import http.client

conn = http.client.HTTPSConnection("api-xyz.com")
payload = ''
headers = {}
conn.request("GET", "/trueid/mobilelegends/?id=366409793&zone=9707", payload, headers)
res = conn.getresponse()
data = res.read()
print(data.decode("utf-8"))

File: MEI/Hasil/CODING/ID GAME/ML.js
OkHttpClient client = new OkHttpClient().newBuilder()
  .build();
MediaType mediaType = MediaType.parse("text/plain");
RequestBody body = RequestBody.create(mediaType, "");
Request request = new Request.Builder()
  .url("https://api-xyz.com/trueid/mobilelegends/?id=960207848&zone=12821")
  .method("GET", body)
  .build();
Response response = client.newCall(request).execute();